Python/Image_Processing/README.md

```diff
@@ -0,0 +1,9 @@
+
+# #6500 [Python] : Image Processing using Python
+
+- Image Processing is the method through which, some meaningful operations can be performed on the Image using tools or languages.
+
+- We use Python here for the Image processing here
+
+- Contributed by [burpcat](https://www.github.com/burpcat)
```

Comment:
1. Please remove this line, Contributors are not supposed to add their credentials in their contributions.
2. Make the README more elaborate regarding the topic.

Reply:
I made the changes, please review